### PR TITLE
Allow admins to manage search results

### PR DESCRIPTION
**Pull request by tijmenb**
This commit adds a page to manage search results. The user can enter a URL (for example for `/oil-and-gas` or `https://www.gov.uk/oil-and-gas` and see the information contained in elasticsearch for this link. The user can also delete the search result.

## Screenshots

You can look up the search result by URL or path.

Whatever is returned by rummager is displayed as nicely formatted nested tables.

After the user clicks the big red delete button, the search result is removed and the user is redirected back. Because of re-indexing lag the search result is still there.

By refreshing the user can verify that the search result has indeed disappeared from the index.

Trello: https://trello.com/c/0Jybv3yh/276-remove-item-from-search-index
PR in rummager to add endpoints: https://github.com/alphagov/rummager/pull/475

### CODE REVIEW

**Comment by KushalP** [comment]
The code for this looks fine, but I wonder whether we should be noisier about deleting documents from the index? /cc @alextea @markhurrell

**Comment by tijmenb** [comment]
Updated the removal button with extra context:

screen shot 2015-08-07 at 10 51 55

**Comment by tijmenb** [comment]
screen shot 2015-08-10 at 14 58 59

Amended text again. We'll tune the copy after deploy if anything is unclear.

File: app/controllers/results_controller.rb [hunk 1]
```diff
@@ -0,0 +1,29 @@
+class ResultsController < ApplicationController
+  def index
+  end
+
+  def show
+    @path = params[:base_path]
+    @path = URI.parse(@path).path if @path.starts_with?('http')
```
Comment (rboulton) [inline app/controllers/results_controller.rb]: Some documents in the index actually start their link field with "http"; eg, http://www.nhs.uk/

I think this parsing should either be removed, or only apply if the path starts with https://www.gov.uk

**Pull request merged**